app/templates/suppliers/companies_house_number.html

```diff
@@ -28,12 +28,13 @@
 
   <div class="grid-row">
     <div class="column-two-thirds">
-      <div class="hint">
-        <p>This is a unique 8-character number given to your organisation by Companies House.</p>
-        <p><a href="https://beta.companieshouse.gov.uk/">Visit Companies House to get your number</a></p>
-      </div>
       <form method="POST" action="{{ url_for('.companies_house_number') }}">
         <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
+        {% set question_advice =
+            'This is a unique 8-character number given to your organisation by Companies House.
+            \n\n<a href="https://beta.companieshouse.gov.uk/">Visit Companies House to get your number</a>
```

Comment:
I'd quite like to hear an opinion from @tombye - not because I think he'll agree with me, but just because I've not seen this in templates before and he's likely to have a sensible viewpoint on it.